erizo/src/erizo/WebRtcConnection.cpp

```diff
@@ -737,51 +734,34 @@ void WebRtcConnection::write(std::shared_ptr<dataPacket> packet) {
   transport->write(packet->data, packet->length);
 }
 
-void WebRtcConnection::sendLoop() {
+void WebRtcConnection::sendPacket(std::shared_ptr<dataPacket> p) {
+  if (!sending_) {
```

Comment:
:+1: to this
Just wanted to emphasize we need to be on the lookout for the possible side effects the new worker system brings. This is a good example.